File: All Codes/URI_1018.cpp
#include <iostream>

using namespace std;

int main()
{
	int N;
	int a[7]={100,50,20,10,5,2,1};
	int temp;
	
	cin >> N;
	cout << N << endl;

	for (int i = 0; i < 7; i++) {
		temp = N / a[i];
		N = N % a[i];
		cout << temp << " nota(s) de R$ " << a[i] << ",00" << endl;
	}
    return 0;
}

File: All Codes/URI_1018.java
import java.util.Scanner;

/**
 * IMPORTANT: 
 *      O nome da classe deve ser "Main" para que a sua solução execute
 *      Class name must be "Main" for your solution to execute
 *      El nombre de la clase debe ser "Main" para que su solución ejecutar
 */
public class URI_1018 {

    public static void main(String[] args) {
        try (Scanner input = new Scanner(System.in)) {
            int[] arr = {100, 50, 20, 10, 5, 2, 1};
            int temp = 0;
            int N = input.nextInt();

            System.out.printf("%d\n", N);


            for (int i = 0; i < arr.length; i++) {
                temp = N / arr[i];
                N %= arr[i];
                System.out.printf("%d nota(s) de R$ %d,00\n", temp, arr[i]);
            }
        }
    }
}